### PR TITLE
Add comparison tab to results view, including enrichments tabs

### PR DESCRIPTION
**Pull request by adamabeshouse**
TO-DO:
- [x] remove old enrichments tabs code
- [x] add tests

### CODE REVIEW

**Comment by jjgao** [comment]
Cool! Thanks, @adamabeshouse. It looks great 👍 

@schultzn @tmazor @cBioPortal/product do you think we should move Survival into Comparison or better leave it separate?

**Comment by tmazor** [comment]
@jjgao Survival does seem to make sense within the Comparison tab, but if we make that change, we may need to help users learn to look there. I have a slight preference to put it in Comparison, but really I can go either way on that decision. @schultzn ?

File: src/pages/resultsView/survival/SurvivalTab.tsx [hunk 1]
```diff
@@ -117,7 +117,11 @@ export default class SurvivalTab extends React.Component<ISurvivalTabProps, {}>
                             yLabelTooltip="Survival estimate"
                             xLabelWithEventTooltip="Time of death"
                             xLabelWithoutEventTooltip="Time of last observation"
-                            fileName="Overall_Survival" />
+                            fileName="Overall_Survival"
```
Comment (alisman) [inline src/pages/resultsView/survival/SurvivalTab.tsx]: do we need this fine anymore?

**Comment by alisman** [comment]
@adamabeshouse just want to ask how session is dealt with in context of resultsview.   We don't have comparison session anymore here, right?  it's just whatever is defined by results view as the two groups?

**Comment by adamabeshouse** [comment]
@alisman right, the sessionId stuff is not put into the superclass ComparisonStore because it is only for GroupComparisonStore. in the results view the groups are just altered/unaltered and we disabled group creation from the overlap tab

**Comment by inodb** [comment]
@adamabeshouse could you maybe add some screenshots or a GIF to this PR? Useful for the release notes + news item.

**Pull request merged**